.gitignore

```diff
@@ -38,11 +38,11 @@
 # End of https://www.gitignore.io/api/c++
 
 # custom ignores
-cmake-build-*
-build
+.idea/
```

Comment:
`.idea` should not be in the project `.gitignore`, and should go in your local `.gitignore` if you are developing with CLion.